src/views/ConnectionBrowser.ts

```diff
@@ -268,7 +261,7 @@ class ConnectionBrowser implements vscode.TreeDataProvider<ServerItem> {
 
   async getChildren(): Promise<ServerItem[]> {
     const lastConnection = GlobalStorage.get().getLastConnections()?.[0];
-    return (GlobalConfiguration.get<ConnectionData[]>(`connections`) || [])
+    return (ConnectionManager.getAll() || [])
```

Comment:
`getAll` already returns an array no matter what.
```suggestion
    return ConnectionManager.getAll()
```